openpathsampling/tests/analysis/utils.py

```diff
@@ -0,0 +1,500 @@
+"""
+Tools for generating sequences of steps for analysis tests.
+
+The idea here is to have client code specify things like which snapshot
+index is the shooting point and what partial trajectory from one-way
+shooting should be, then we patch those things and use the internals of the
+mover to generate the real data.
+
+In this way, we use as much of the actual OPS machinery as possible,
+ensuring, for example, that the details we return are correct.
+
+Note that for the classes here, each instance represents a single move
+(single MC step). This is unlike the PathMover objects in OPS, which
+represents an individual type of move that can be reused for many steps.
+"""
+
+import numpy as np
+import random
+try:
+    from unittest.mock import Mock, patch
+except ImportError:
+    from mock import Mock, patch
+
+from openpathsampling.tests.test_helpers import make_1d_traj
+import openpathsampling as paths
+
+def _get_only(iterable, condition, error_msg):
```

Comment:
pep8: 2 blank expected, found 1
```suggestion


def _get_only(iterable, condition, error_msg):
```